package.json

```diff
@@ -136,13 +136,14 @@
     "@scure/base": "1.1.1",
     "@scure/bip32": "1.1.5",
     "@scure/bip39": "1.1.1",
+    "@scure/btc-signer": "0.5.0",
     "@segment/analytics-next": "1.49.2",
     "@sentry/react": "7.35.0",
     "@sentry/tracing": "7.35.0",
     "@stacks/auth": "6.1.1",
     "@stacks/blockchain-api-client": "6.3.4",
     "@stacks/common": "6.0.0",
-    "@stacks/connect": "7.1.0",
+    "@stacks/connect": "5.0.6-alpha.9d9606f.0",
```

Comment:
Stacks packages need to updated here, and removed from resolutions, once the Connect PR is merged/released.

Reply:
https://github.com/hirosystems/connect/pull/310